pkgs/development/embedded/arduino/arduino-cli/default.nix

```diff
@@ -68,7 +68,7 @@ let
       mainProgram = "arduino-cli";
       changelog = "https://github.com/arduino/arduino-cli/releases/tag/${version}";
       license = licenses.gpl3Only;
```

Comment:
Maybe we need to add `asl20` to licenses: https://github.com/arduino/arduino-cli/pull/2653

Reply:
Oops, I was so focused at looking for dependency changes that I missed this. Done.